frb_codegen/src/binary/commands.rs

```diff
@@ -221,10 +221,22 @@ pub(crate) struct CreateCommandArgs {
 
 #[derive(Debug, Args)]
 pub(crate) struct IntegrateCommandArgs {
-    /// Generate code related to integration test
+    /// Do NOT generate code related to lib/example etc.
+    #[arg(long)]
+    pub(crate) no_enable_overlay_lib: bool,
```

Comment:
nit: would be great to change the name a bit, since "overlay lib" may not be super clear